src/components/TransactionFlow/displays/TokenTransferTable.tsx

```diff
@@ -0,0 +1,169 @@
+import { useState } from 'react';
+
+import { toChecksumAddress } from '@mycrypto/wallets';
+import styled from 'styled-components';
+
+import { Amount, Box, Icon } from '@components';
+import { EthAddress } from '@components/EthAddress';
+import { Text } from '@components/NewTypography'
+import { getFiat } from '@config/fiats';
+import { BREAK_POINTS, COLORS, SPACING } from '@theme';
+import { translateRaw } from '@translations';
+import { ISettings } from '@types';
+import { bigify, convertToFiat } from '@utils';
+
+import { ITxTransferEvent } from '../TxReceipt';
+
+interface Props {
+  valueTransfers: ITxTransferEvent[];
+  settings: ISettings;
+}
+
+const ERC20Box = styled(Box)`
+  padding: 12px;
+  margin: ${SPACING.MD};
+  background-color: ${COLORS.GREY_LIGHTEST};
+`
+
+const Table = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+`;
+
+const Body = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  @media screen and (max-width: ${BREAK_POINTS.SCREEN_SM}) {
+    border-radius: 1.3px;
+    & > *:first-child {
+      border-top: 1px solid ${COLORS.GREY_ATHENS};
+    }
+  }
+`;
+
+const Row = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  min-height: 60px;
+  width: 100%;
+  gap: ${SPACING.XS};
+  & > *:last-child {
+    flex: 1;
+  }
+  @media screen and (max-width: ${BREAK_POINTS.SCREEN_SM}) {
+    flex-direction: column;
+    align-items: flex-start;
+    position: relative;
+    width: 100%;
+    border-bottom: 1px solid ${COLORS.GREY_ATHENS};
+    padding: ${SPACING.SM} 0 ${SPACING.SM} 0;
+    & > *:last-child {
+      align-self: flex-end;
+      flex: 1;
+      border-bottom: 0px;
+    }
+  }
+`;
+
+const SText = styled(Text)`
+  margin: ${SPACING.NONE} ${SPACING.XS} ${SPACING.NONE} ${SPACING.NONE};
+`
+
+const STransferAmounts = styled(Text)`
+  /* Night Mode/ Accent */
+  padding: 1px 4px;
+  background: #8F8F8F;
+  font-weight: bold;
+  border-radius: 5px;
+  color: white;
+  margin: ${SPACING.NONE};
+`
+
+const LabelBox = styled(Box)`
+  gap: ${SPACING.XS};
+`
+
+const SMoreIconWrapper = styled.div`
```

Comment:
Can we use shorthand props on the components instead of styled components for all of the styling?

Reply:
> Can we use shorthand props on the components instead of styled components for all of the styling?

Why?